docs/wiki/framework/wardrobe-framework.md

```diff
@@ -0,0 +1,300 @@
+---
+layout: wiki
+title: Wardrobe Framework
+description: Explains the wardrobe system and how to make a compat.
+group: framework
+order: 5
+parent: wiki
+mod: ace
+version:
+  major: 3
+  minor: 20
+  patch: 0
+---
+
+
+## 1. Overview
+
+The Wardrobe Addon gives the player the opportunity to change/modify their current wearables into different variants.
+
+For example, if a uniform has a normal variant and a "Rolled-Up Sleeves" variant, the player will be able to use ether an ACE Self Interaction or the CBA Context Menu in the Inventory.
+
+Since there is no reliable, common pattern in terms of class inheritence, not even within the same DLC, each possbile variant has to be defined individually within the classes config properties.
```

Comment:
```suggestion
Since there is no reliable, common pattern in terms of class inheritance, not even within the same DLC, each possible variant has to be defined individually within the classes' config properties.
```